react/index.js

```diff
@@ -0,0 +1,61 @@
+import { Component } from 'react'
+
+import { Helmet } from 'render'
+
+import { Pixel } from 'vtex.store/PixelContext'
+
+const APP_LOCATOR = 'vtex.adwords'
+
+class Adwords extends Component {
+  constructor(props) {
+    super(props)
+    this.props.subscribe(this)
+  }
+  
+  gtag(data) {
+    dataLayer.push(data)
+  }
+  
+  get adwordsID() {
+    const { adwordsID } = this.context.getSettings(APP_LOCATOR) || {} 
+    if (!adwordsID) {
+      console.warn("Warning: The Google Adwords ID isn't set on your environment.")
+    }
+    return adwordsID 
+  }
+
+  productView = (event) => {
+    console.log(">>> Adwords >>> Product view: ", event)
+    const skuId = event.products[0].id
```

Comment:
I think this can cause an exception, so you should check if the products isn't undefined before access the element.